kin-sdk-core/src/main/java/kin/sdk/core/TransactionSender.java

```diff
@@ -0,0 +1,124 @@
+package kin.sdk.core;
+
+
+import android.support.annotation.NonNull;
+import java.io.IOException;
+import java.math.BigDecimal;
+import kin.sdk.core.exception.OperationFailedException;
+import kin.sdk.core.exception.PassphraseException;
+import org.stellar.sdk.Asset;
+import org.stellar.sdk.KeyPair;
+import org.stellar.sdk.PaymentOperation;
+import org.stellar.sdk.Server;
+import org.stellar.sdk.Transaction;
+import org.stellar.sdk.responses.AccountResponse;
+import org.stellar.sdk.responses.SubmitTransactionResponse;
+
+class TransactionSender {
+
+    private final Server server;
+    private final KeyStore keyStore;
+    private final KeyPair issuer;
+    private final Asset kinAsset;
+
+    TransactionSender(Server server, KeyStore keyStore, KeyPair issuer, Asset kinAsset) {
+        this.server = server;
+        this.keyStore = keyStore;
+        this.issuer = issuer;
+        this.kinAsset = kinAsset;
```

Comment:
should assets include the issuer as part of the asset information? https://www.stellar.org/developers/guides/concepts/assets.html